packages/salesforcedx-vscode-soql/src/editor/soqlEditorProvider.ts

```diff
@@ -68,13 +59,14 @@ export class SOQLEditorProvider implements vscode.CustomTextEditorProvider {
   }
 
   private getWebViewContent(webview: vscode.Webview): string {
-    const pathToLwcDist = path.join(
-      this.extensionContext.extensionPath,
-      SOQL_BUILDER_UI_PATH
+    const soqlBuilderWebAssetsPathParam: string[] =
+      this.extensionContext.extension.packageJSON.soqlBuilderWebAssetsPath;
+    const soqlBuilderUIModule = this.extensionContext.asAbsolutePath(
+      path.join(...soqlBuilderWebAssetsPathParam, DIST_FOLDER)
     );
-    const pathToHtml = path.join(pathToLwcDist, HTML_FILE);
+    const pathToHtml = path.join(soqlBuilderUIModule, HTML_FILE);
     let html = fs.readFileSync(pathToHtml).toString();
-    html = HtmlUtils.transformHtml(html, pathToLwcDist, webview);
+    html = HtmlUtils.transformHtml(html, soqlBuilderUIModule, webview);
```

Comment:
Just to be that guy...do these have unit test coverage?

Reply:
Sadly there's only some unit test coverage for all soql stuff, and very minimal

Reply:
I'm currently working on E2E Tests for this though.

Reply:
I took a stab at creating a jest file for this...

Reply:
tytytytyty